html/templates/header.html

```diff
@@ -35,16 +35,16 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<font size='2'><b>Current Headcoders:</b> <a href='https://forums.yogstation.net/index.php?threads/admin-list-and-rankings.10821/#post-127765'>-Click Here-</a><br></font>
-			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation-TG/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Coders:</b> Nichlas0010, ThatLing, GrayRachnid, AsV9, Kmc2000, Moja10, Cruix, X-TheDark, Oisin100, TehFlaminTaco, Kn0ss0s, Super3222, ShadowDeath6, Time-Green, Ephemeralis, Alek2ander, FluffySurvivor, Nirnael, Utahclock, Macelarius, ToGWTF, Adam El-Tablawy, Johncena1469, McDonald072, ArgyleSweatshop, Altoids<br></font> 
-			<font size='2'><b>Spriters:</b> Partheo, Kmc2000, Andy, Silvaplex, Dagdamnit, Maya/Larissa<br></font>
-			<font size='2'><b>Mappers:</b> Nichlas0010, GrayRachnid, Matskuman5, Utahclock, Stealthkibbler, MayhemSailor, Yewmi<br></font>
+			<font size='2'><b>Current Headcoders:</b>AsV9<br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/yogstation13/Yogstation/graphs/contributors'>-Click Here-</a><br></font>
+			<font size='2'><b>Maintainers:</b> Alexkar598, Theos, Fluffe9911, Ktlwjec, McDonald072, Monster860, ThatLing, AshCorr, Partheo, Identification<br></font> 
+			<font size='2'><b>Spriters:</b> Missatessatessy, Kmc2000, Identification, r3d, Meeeeep7, Keekenox, Partheo<br></font>
+			<font size='2'><b>Mappers:</b> Ktlwjec, Monster860, MayhemSailor, DotLyna<br></font>
```

Comment:
The spriters and mappers section is pretty useless as there's no set requirement to be a "mapper" or "spriter".

Reply:
Didn't DotLyna resign?

Reply:
wait meeeeep7 sprites ive never seen any

Reply:
> Didn't DotLyna resign?

according to ktl. no

Reply:
> wait meeeeep7 sprites ive never seen any

he does